services/blockchain-connector/shared/utils/download.js

```diff
@@ -100,9 +103,43 @@ const downloadFile = (url, dirPath) => new Promise((resolve, reject) => {
 	});
 });
 
+const verifyFileChecksum = async (filePath, checksumPath) => {
```

Comment:
Please add an `info` log in `downloadFile` to state that a file is being downloaded for a specific URL. Kindly ensure that other corresponding methods have the same as well.

Reply:
Applied